packages/flutter/lib/src/widgets/virtual_viewport.dart

```diff
@@ -162,7 +182,12 @@ abstract class VirtualViewportElement<T extends VirtualViewport> extends RenderO
   }
 }
 
-abstract class VirtualViewportIterableMixin extends VirtualViewport {
+/// A VirtualViewport that represents its children using [Iterable<Widget>].
+///
+/// The iterator is advanced just far enough to obtain widgets for the children
+/// that need to be materialized.
+abstract class VirtualViewportIterable extends VirtualViewport {
```

Comment:
Should this be "IterableVirtualViewport" or "IteratingVirtualViewport"? "VirtualViewportIterable" sounds like it's an Iterable, which it isn't.